.github/workflows/tests.yml

```
@@ -1,6 +1,11 @@
 name: neofs-sdk-go tests
 
 on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '*.md'
```

```suggestion
      - '**/*.md'
```
not sure it changes anything to us now but this it how it is done below

---

Fixed.